AdblockPlusSafariExtension/AdblockPlus+Extension.m

```diff
@@ -17,41 +17,35 @@
 
 #import "AdblockPlus+Extension.h"
 #import "AdblockPlus+Parsing.h"
+#import "NSDictionary+FilterList.h"
 
 @implementation AdblockPlus (Extension)
 
 - (NSURL *__nullable)activeFilterListsURL
 {
-  NSString *fileName;
+  NSString *filename;
```

Comment:
Why the rename, isn't `fileName` how we normally capitalise this in the code base? Assuming Apple's coding style doesn't cover this case, I suggest we stay consistent with the rest of the project.